coalib/coala_main.py

```diff
@@ -153,6 +154,12 @@ def run_coala(console_printer=None,
         if not sections['cli'].get('disable_caching', False):
             cache = FileCache(None, os.getcwd(), flush_cache)
 
+        if len(targets) != 0:
+            new_sections = collections.OrderedDict()
```

Comment:
Do above `from collections import OrderedDict`, than you don't have to reference the module here.